codegen-renderers/src/main/kotlin/io/vrap/codegen/languages/java/file/producers/JavaFileProducer.kt

```diff
@@ -0,0 +1,40 @@
+package io.vrap.codegen.languages.java.file.producers
+
+import io.vrap.rmf.codegen.io.TemplateFile
+import io.vrap.rmf.codegen.rendring.FileProducer
+import io.vrap.rmf.codegen.rendring.utils.escapeAll
+import io.vrap.rmf.codegen.rendring.utils.keepIndentation
+
+class JavaFileProducer : FileProducer {
+
+    override fun produceFiles(): List<TemplateFile> {
+        return listOf(produceSphereRequestInterface())
+    }
+
+    private fun produceSphereRequestInterface() : TemplateFile {
+        val content = """
+            |package com.commercetools.importer.commands;
+            |
+            |import javax.annotation.Nullable;
+            |import io.sphere.sdk.http.HttpResponse;
+            |import io.sphere.sdk.client.HttpRequestIntent;
+            |
+            |public interface SphereRequest<T> {
```

Comment:
We should rename this to may be `ApiRequest`, but I'm fine to call it `SphereRequest` to stay backwards compatible for now 😉